### DISCUSSION TITLE
SMA connector

### DISCUSSION THREAD

**Discussion by JasiuBEL** [PE5PVB/SI4684-DAB-Receiver#7] [General]
Hey Sjef,

I'm at the stage of ordering components for the radio. I noticed that there is a different SMA connector in the video and a different one in the BOM files (in the project). It would be nice if you would post the BOM with all the parts that YOU used for the project with manufacturer or shop part numbers. It would be easier to complete in the shop. For example, I'm looking for a barrel jack and analysing if this one is ok, will it fit the chassis that is in the project.

I have a few boards with SMD circuits and components fitted, no THT components, that I can sell, if anyone is interested I would be happy to resell.

**Comment by PE5PVB** [comment]
It's just a standard SMA connector with angle, like https://www.tinytronics.nl/nl/kabels-en-connectoren/connectoren/overige/sma-pcb-connector-female-haaks
